examples/javascript.pegjs

```diff
@@ -303,12 +303,12 @@ SingleEscapeCharacter
   = "'"
   / '"'
   / "\\"
-  / "b"  { return "\b"; }
-  / "f"  { return "\f"; }
-  / "n"  { return "\n"; }
-  / "r"  { return "\r"; }
-  / "t"  { return "\t"; }
-  / "v"  { return "\v"; }
+  / "b"%"\b"
+  / "f"%"\f"
+  / "n"%"\n"
+  / "r"%"\r"
+  / "t"%"\t"
+  / "v"%"\v"
```

Comment:
I don't find this easier to read or maintain.  Are there other examples you think might benefit from this approach?  Maybe add a new example that shows the full power?

Reply:
Agree. In any case it is better to use `->` or `=>`

Reply:
Oh, if it was `=>` I might be able to get excited about this.

Reply:
Particularly if we also added `&=>` and `!=>` at the same time...